pydm/data_plugins/calc_plugin.py

```diff
@@ -241,6 +241,13 @@ def receive_new_data(self, data):
             logger.debug('Connection was not available yet for calc.')
         try:
             val = data.get('value')
+            if isinstance(val, np.generic):
+                if np.issubdtype(type(val), np.floating):
```

Comment:
How about https://numpy.org/doc/stable/reference/generated/numpy.ndarray.item.html ?

Reply:
Yeah, works for me.  Let's just drop this on the floor then. ;-)